src/ppx_deriving_protobuf.cppo.ml

```diff
@@ -94,7 +92,7 @@ let filter_map f lst =
   List.map f lst |> filter []
 
 let string_of_lident lid =
-  String.concat "." (Longident.flatten lid)
+  String.concat "." (Longident.flatten_exn lid)
```

Comment:
Could you use this helper function in the many other places that (you had to change and) use `String.concat "."` in the same way?

Reply:
done